cristin-import/src/test/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumerTest.java

```diff
@@ -301,6 +308,33 @@ public void savesFileInInputFolderErrorExceptionNameInputFileLocationInputFileWh
         assertThat(actualErrorFile, is(not(nullValue())));
     }
 
+    @Test
+    public void handleRequestThrowsExceptionWhenInputContainsUnknownProperty() throws JsonProcessingException {
+        String input = IoUtils.stringFromResources(Path.of("cristin_entry_with_unknown_property.json"));
+
+        AwsEventBridgeEvent<FileContentsEvent<Identifiable>> event = parseEventAsIdentifieableObject(input);
+        Executable action = () -> handler.handleRequest(stringToStream(input), outputStream, CONTEXT);
+        RuntimeException exception = assertThrows(RuntimeException.class, action);
+
+        UnixPath expectedFilePath = constructExpectedFilePathForEntryWithUnkownFields(event, exception);
+
+        S3Driver s3Driver = new S3Driver(s3Client, NOT_IMPORTANT);
+        String file = s3Driver.getFile(expectedFilePath);
+
+        assertThat(file, is(not(emptyString())));
+        assertThat(file, containsString(UNKNOWN_PROPERTY_NAME_IN_RESOURCE_FILE_WITH_UNKNOWN_PROPERTY));
+    }
+
+    // This test will fail until we have mapped all properties in the resource file or we
+    // have removed (from the resource and the import) the ones that we do not want to keep.
+    @Disabled
```

Comment:
Test for helping spotting unknown fields. The alternative would be to run the job in AWS multiple times (once for each unknown field), or create some complex logic that lists all unmapped fields. With this test one can verify that they have mapped all fields.